data-raw/jup_pkgs.R

```diff
@@ -0,0 +1,22 @@
+## code to prepare `jup_pkgs` dataset goes here
+
+jup_pkgs <- c(
+  "adjustbox",
+  "caption",
+  "collectbox",
+  "enumitem",
+  "environ",
+  "eurosym",
+  "jknapltx",
+  "parskip",
+  "pgf",
+  "rsfs",
+  "tcolorbox",
+  "titling",
+  "trimspaces",
+  "ucs",
+  "ulem",
+  "upquote"
+)
+
+usethis::use_data(jup_pkgs, overwrite = TRUE)
```

Comment:
I don't want to introduce the dependency on **usethis** only for the purpose of saving a simple vector to `.rda` and documenting it. I prefer not exporting or documenting `jup_pkgs` here. That means users will need to use `:::`, which I think is fine. This is not a super important object anyway.

That said, it will be good to briefly mention it on the TinyTeX documentation site, e.g. after FAQ 12: https://yihui.org/tinytex/faq/

An alternative approach is like this: https://github.com/yihui/tinytex/blob/b84fd550ffb72a8f20d323144fe269f4a5173c9a/R/install.R#L342-L346 i.e. save the list of packages under `tools/pkgs-jupyter.txt`, so both the R function and the shell script can read it.

Reply:
I was curious as to whether adding `usethis::use_data_raw()` would mean that it needed to be added as a dev dependency. So I queried Allison Hill on this. She said that "The scripts in data_raw/ are for you. They are your way of creating the package but that directory is added to your `.Rbuildignore` when you set it up with `usethis`" (see Twitter conversation [here](https://twitter.com/TiffanyTimbers/status/1288949392112672768)). So in general it seems that technically this does not make `usethis` a dev dependency. However, I am happy to defer to your preferred way to do this. Another advantage of doing something like `install_yihui_pkgs` with a the list of packages in `tools/pkgs-jupyter.txt` is that, as you mentioned, the shell script can read it.

Once you confirm this is for sure the way you want to go, I will change to PR to reflect it.

Reply:
Yes, please store the list of packages in `tools/pkgs-jupyter.txt`, and remove the dependency on **usethis**. Thank you!